notebooks/demo.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>I assume we have a FAQ doc within the doc search. wdyt if we exclude the faq doc and include other docs which might have some answers related to faq's? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Adding a section on non FAQ included version.